nasa-gibs/worldview
# WV-3372 Earthdata Search Logo Change

## Description
This change modifies the "logo" of Earthdata Search to be in-line with new HDS designs.

## How To Test
1. `git checkout wv-3372-earthdata-search-logo`
2. `npm ci`
3. `npm run watch`
4. Open a fresh instance of Worldview
5. Open the Data tab in the sidebar on the left side of WV, then click the "DOWNLOAD VIA EARTHDATA SEARCH" button at the bottom.
6. Verify that the Earthdata Search logo at the top of the window that appears matches the new designs
